PhilanthropyDataCommons/front-end
# Add Organization Table Pages

This PR adds a view for the Organizations table. It is similar in structure to the current view for the '/proposals' route, but does not employ base-fields. 


### Testing
- run `npm run start` in the front-end repository
- run `npm run start` in the service repository
- populate organizations table
- on the front-end in your browser, go to `http://localhost:3000:organizations`
- click on an organization and verify that it redirects to the chosen organization's detail page

Closes #596
